### ISSUE TITLE
referencing of service by class name in key of service list

### ISSUE THREAD

**Issue by vvoody-** [mesour-intellij-plugins/neon-nette-issues#13]
Plugin v1.0.0 is not expexting `@` sign at start of service key. 

![image](https://user-images.githubusercontent.com/2100111/149751542-eb65f89e-a355-467e-a612-342bfc66181e.png)

Nette DI allows referencing services by its class name with `@` prefix:

```neon
three: @\Lorem
one:
	factory: Lorem(@\Ipsum)
two:
	factory: Ipsum(1)
	setup:
		- @\Ipsum::foo()
four: @\Lorem
@\Ipsum:
	arguments: [2]
```
source https://github.com/nette/di/blob/master/tests/DI/Compiler.services.byClass.phpt#L54

**Comment by mesour** [comment]
@vvoody- references for class working, but not in keys. I will extend parser also for keys

**Comment by mesour** [comment]
Fixed in 1.0.2
